support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -95,6 +127,60 @@ function getMvtId(): number {
 	return mvtId;
 }
 
+function getParticipations(
+	abTests: Tests,
+	mvtId: number,
+	country: IsoCountry,
+	countryGroupId: CountryGroupId,
+	acquisitionDataTests?: AcquisitionABTest[],
+): Participations {
+	const participations: Participations = {};
```

Comment:
This function has been moved up in the file to reflect its importance. It's also changed slighty due to renaming `userInTest` to `getUserParticipation` and the new return types of `getUserParticipation` and `assignUserToVariant`. I've also fully commited to `if` guards e.g

```ts
if (condition) {
  return;
}
```

Which has removed all nesting. I think this makes it easier to read